### DISCUSSION TITLE
Divider分割线  如何自定义成这样的样式

### DISCUSSION THREAD

**Discussion by hihuangwei** [ant-design/ant-design#30982] [Q&A]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

![QQ浏览器截图20210614211503](https://user-images.githubusercontent.com/3166799/121898253-ca026180-cd55-11eb-896e-4b23e8ee5c3c.png)

**Comment by xrkffgg** [comment]
我记得有个 less 变量可以控制文字的距离，其实就是改变了 css

而且你看到这个例子的话，直接 F12 看看样式就行

其实纯用 div 也不难
